app/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "mobilecoin-wallet-v1.2.0",
+  "name": "mobilecoin-wallet",
   "productName": "MobileCoin Wallet",
-  "version": "v1.2.0",
+  "version": "1.2.0",
```

Comment:
Are you sure about removing the "v"? I think it was used for (and requested by) the autoupdater. I'd recommend testing if the autoupdater still works after removing the "v".